pensieve/analysis.py

```diff
@@ -180,7 +169,7 @@ def run(self, current_date: datetime, dry_run: bool):
             res_table_name = self._table_name(period.value, window)
 
             sql = exp.build_query(
-                getattr(self.config.metrics, period.adjective),
+                self.config.metrics[period],
```

Comment:
We'll need to check if there actually exists a metric for the `period` in `metrics`

Reply:
My first reaction was "eh, what if we just force it to be a `collections.defaultdict`..." but that's probably too clever.

Reply:
The fix here is actually to iterate as `for period in self.config.metrics:` instead of `for period in AnalysisPeriod:` -- no need to build a query for an interval we don't care about

Reply:
Sweet! 👍